### ISSUE TITLE
Check if post-update script exists before executing it

### ISSUE THREAD

**Issue by c-edw** [clearlinux/swupd-client#824]
Towards the end of building an image with mixer, I get a small non-fatal error. The referenced path looks a bit weird. Apparently this is normal.

I suggested that the a check should be made for whether this path exists before deleting it.

```
$ sudo mixer build image
...
Inspected 8386 files
  8379 files were missing
    8379 of 8379 missing files were replaced
    0 of 8379 missing files were not replaced
Calling post-update helper scripts.
sh: /tmp/ister-latest-6uuayzoc///usr/bin/update-helper: No such file or directory
Fix successful
Cleaning up
Successful installation
```
Thanks.

Originally filed here https://github.com/bryteise/ister/issues/161#issue-410016423.

**Comment by otaviobp** [comment]
Same fix used by #795

**Comment by otaviobp** [comment]
It's easy to reproduce that in swupd by running:

    sudo swupd verify --install --path custom_path -m latest
